package.json

```diff
@@ -35,7 +35,7 @@
     "npm-run-all": "^4.1.5",
     "qs": "^6.10.3",
     "react": "^17.0.1",
-    "react18-bootstrap-sweetalert": "^5.2.2",
+    "react-bootstrap-sweetalert": "^5.2.0",
```

Comment:
I think this is an oversight from my side. The one with react18... is specifically tailored for react 18, therefore I think it might be best to keep the old one, when we are using react: ^17.0.1.